visma/functions/variable.py

```diff
@@ -77,3 +77,177 @@ def integrate(self, wrtVar):
 
     def calculate(self, val):
         return self.coefficient * ((val**(self.power)))
+
+    def __radd__(self, other):
+        return self + other
+
+    def __add__(self, other):
+        from visma.functions.constant import Constant
+        if isinstance(other, Variable):
+            sortedValuesSelf = self.value.sort()
+            sortedValuesOther = other.value.sort()
+            if (self.power == other.power) & (sortedValuesSelf == sortedValuesOther):
+                if self.before == '-':
+                    self.coefficient -= other.coefficient
+                else:
+                    self.coefficient += other.coefficient
+                result = Variable()
```

Comment:
Since `self` is being returned no need to use `result`. Also return 0 if `self.coefficient == 0`.